build-system/compile/single-pass.js

```diff
@@ -324,7 +322,11 @@ exports.getGraph = function(entryModules, config) {
     // directly and which we don't want to apply during deps finding.
     .transform(babelify, {
       compact: false,
```

Comment:
I think the `compact` option needs to be inside the `overrides` array.

Reply:
In fact, you can just delete it because it's already present in `babel.config.js`:

https://github.com/ampproject/amphtml/blob/3440444dcd7721af9daaef07cfbc30a4f0905e62/babel.config.js#L67